tasks/rss-mentor-dashboard.md

```diff
@@ -1,77 +1,83 @@
+    - connects (merges, joins) data from [google drive folder](https://drive.google.com/open?id=1ULj8KjnNNCgUdGunQ1TY00dNbCsqAsHW)
+      - about mentor's students
+      - about students' marks
+    - gets json file with needed data
+    - loads data on html page
+  - with help of React and React component draws dashboard, using resulting json
+  - build with webpack and deploy on github pages
+  - Pull Request should contain
+    - nodejs script, instruction how to use it
+    - resulting json
+    - js file with React component
+    - html page, which loads React, js file with React, json file with data, which renders React component
+  - post link to github page + PR [into form](https://docs.google.com/forms/d/e/1FAIpQLSctLqyRXrpdgC1vybGjGZoiYwZUUAf6_hSD-gPTZMORS_E1kQ/viewform)
 
-### Критерий оценки:
-- Максимальный бал за задание - **140**
-- Реализована генераций json файла - **+25**
-- Реализовано подключение json файла на страницу и вывод списка студентов ментора - **+10**
-- Реализовано подключение json файла на страницу и вывод списка тасков ментора - **+10**
-- Реализовано подключение json файла на страницу и выбор ментора из контрола выбора менторов - **+10**
-- Реализована подгрузка выбранного ментора из локал стораджа - **+5**
-- Реализована цветовая индикация **+10**
-- Требования к репозиторию, коммитам и PR выполены **+5 баллов**
-- Тесты покрывают следующую фунциональность 
-  - функции мержа данных - **+10**
-  - правильность генерирования итогового json **+10**
-  - факт рендера данных - **+5**
-- Реализован логин и выбор ментора по github аккаунту **+30**
-- до **-50** баллов за нарушение [stage2-tasks-requirements](https://github.com/rolling-scopes-school/docs/blob/master/stage2-tasks-requirements.md)
-- если сделан сервер, который раз в пару минут проверяет гугл документ и умеет отдавать свежий json на github pages - ***+10***
 
-## Кто выполянет проверку задания?
-Ваш ментор.
+### Requirements for commits, PR, repository
+[Common requirements from stage 2](https://github.com/rolling-scopes-school/docs/blob/master/stage2-tasks-requirements.md)
 
-## Доп материалы
-https://www.youtube.com/watch?v=YljG0j-9Vko - видео с разбором задания (первый час). Второй час - небольшой лайфкодинг аналогичного задания.
+### Assessment criteria:
+- Maximum score for the task - **140**
+- Generation of json file is implemented - **+25**
+- Loading json file on page and rendering list of mentor's students is implemented - **+10**
+- Loading json file on page and rendering list of mentor's tasks is implemented - **+10**
+- Loading json file on page and selecting mentor from "select mentor" field is implemented - **+10**
+- Loading selected mentor from localStorage is implemented - **+5**
+- Color indication is implemented **+10**
+- Requirements for commits, PR, repository are satisfied **+5 баллов**
+- Tests cover following functionality:
```

Comment:
Tests cover **the** following functionality

Reply:
fixed